src/test/data/com/hotels/bdp/waggledance/mapping/service/impl/YamlFederatedMetaStoreStorageTest/invalid-federation.yml

```diff
@@ -0,0 +1,4 @@
+federated-meta-stores:
```

Comment:
Should probably rename all these yaml files to not have waggledance in the path name and also the corresponding test to refer to them via a path which is more in line with the package names in use for this new library.

Reply:
these slipped through